opensrp-reveal/src/test/java/org/smartregister/reveal/util/FamilyJsonFormUtilsTest.java

```diff
@@ -209,12 +211,13 @@ public void testGetAutoPopulatedJsonEditMemberForm() throws JSONException {
     @Test
     public void testGetAutoPopulatedJsonEditMemberFormStringForEstimateDob() throws JSONException {
         setupMemberForm();
+        DateTime dob = new DateTime().minus(Years.years(37));
+        client.getColumnmaps().put(KEY.DOB, dob.toString());
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unnecessary use of fully qualified name 'KEY.DOB' due to existing static import 'org.smartregister.family.util.DBConstants.KEY.DOB'](https://app.codacy.com/manual/OpenSRP/opensrp-client-reveal/pullRequest?prid=4759359)